account_asset_management/account_asset_invoice.py

```diff
@@ -4,7 +4,7 @@
 #    OpenERP, Open Source Management Solution
 #
 #    Copyright (C) 2010-2012 OpenERP s.a. (<http://openerp.com>).
-#    Copyright (c) 2014 Noviat nv/sa (www.noviat.com). All rights reserved.
+#    Copyright (c) 2014-now Noviat nv/sa (www.noviat.com).
```

Comment:
2015 doesn't make a lot of sense since the code has been made in 2014 (and parts of the account_asset_management module a lot earlier since the work last year was based upon several customer specific changes done on the standard module in the past couple of years.
I'll change it into 2014-2015. 
Is there any concrete OCA guideline on the copyright clause ?